pennylane/capture/explanations.md

```diff
@@ -32,6 +32,106 @@ def my_func(x):
 [a:f32[1] = my_func b]
 ```
 
+# Higher Order Primitives and nested jaxpr
+
+Higher order primitives are essentially function transforms. They are functions that accept other
+functions. Our higher order primitives will be `adjoint`, `ctrl`, `for_loop`, `while_loop`, `cond`, `grad`,
+and `jacobian`.
+
+Jax describes two separate ways of defining higher order derivatives:
+
+1) *On the fly processing*: the primitive binds the function itslf as metadata
+
+2) *Staged processing*: the primitive binds the function's jaxpr as metadata.
+
+Jax also has a [`CallPrimitive`](https://github.com/google/jax/blob/23ad313817f20345c60281fbf727cf4f8dc83181/jax/_src/core.py#L2366)
+but using this seems to be more trouble than its worth so far. Notably, this class is rather private and undocumented.
+
+We will proceed with using *staged processing* for now.
```

Comment:
Curious what the motivation is?

Reply:
1) Mimics catalyst 

2) allows us to compile the subjaxpr

3) Easy to visually check what's where